rcl_action/src/rcl_action/names.c

```diff
@@ -51,6 +57,11 @@ rcl_action_get_cancel_service_name(
 {
   RCL_CHECK_ALLOCATOR_WITH_MSG(&allocator, "allocator is invalid", return RCL_RET_INVALID_ARGUMENT);
   RCL_CHECK_ARGUMENT_FOR_NULL(action_name, RCL_RET_INVALID_ARGUMENT);
+  RCL_CHECK_ARGUMENT_FOR_NULL(action_name, RCL_RET_INVALID_ARGUMENT);
```

Comment:
Redundant.

Reply:
Argh, yeah, fixed in 276aed1dffbb375fa5bb8cde52ddc28fe6e7bc56.